src/GridFS/Bucket.php

```diff
@@ -620,7 +620,7 @@ public function rename($id, $newFilename)
      * @throws StreamException if the file could not be uploaded
      * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
      */
-    public function uploadFromStream($filename, $source, array $options = [])
+    public function uploadFromStream(string $filename, $source, array $options = [])
```

Comment:
Noted that there's no `resource` type hint per https://wiki.php.net/rfc/resource_typehint.